src/transfer.ts

```diff
@@ -2,6 +2,20 @@ import * as fs from 'fs';
 import { App, Editor, FileSystemAdapter, MarkdownView, TFile, TFolder, normalizePath } from 'obsidian';
 import { VaultTransferSettings } from 'settings';
 import { showNotice } from 'utils';
+
+/**
+ * Simple function that remove a part of a path using the settings "removePath"
+ * @param settings {VaultTransferSettings} 
+ * @param path {string}
+ * @returns {string} The path without the parts to remove or the original path, depending on the settings
+ */
+function removePartOfPath(settings: VaultTransferSettings, path: string):string {
+    for (const part of settings.removePath) {
+        path = path.replace(part, "");
+    }
```

Comment:
By using regex, we can remove paths that show up multiple times, and have it be case insensitive.

```suggestion
    for (const part of settings.removePath) {
        path = path.replace(RegExp(part, "gi"), "");
    }
```